homeassistant/components/ezviz/select.py

```diff
@@ -95,3 +122,43 @@ def select_option(self, option: str) -> None:
             raise HomeAssistantError(
                 f"Cannot set Warning sound level for {self.entity_id}"
             ) from err
+
+
+class EzvizBatteryWorkModeSelect(EzvizEntity, SelectEntity):
```

Comment:
Might be cleaner to add a Mixin to the EzvizSelectEntityDescription with the methods (fuctions) used to set the options. Could then perhaps get away with a single EzvizSelect class.

Reply:
@pjbuffard tagging you just in case you missed the comments notification 👀

Reply:
Hello, yes thanks ! I will try to have a look asap but it might not be before April (holiday time 🏖️ )

Reply:
I hope it is better/cleaner now: I added a `EzvizSelectEntityActionBase` to handle real actions for each type of select. So, as requested, there is still only one `EzvizSelect` 😉

Reply:
ping for @RenierM26 and @BaQs 👀